edudoc/guidelines-for-disciplinary-incidents/guidelines-for-disciplinary-incidents.md

```diff
@@ -1,93 +1,91 @@
+  - If you believe that similar incidents may occur in the future and that it would be beneficial for the WIC to document the incident.
+- For extreme situations where competitor safety may be involved, please contact local law enforcement immediately. Once the immediate concerns of safety have been addressed, contact the WIC with all evidence gathered for the WIC to use in their investigation.
 
 ## Theft at Competitions
 
-* The WDC’s preference is for theft (such as stealing puzzles) to be investigated and resolved locally when possible, as the local Delegates can often resolve these cases better than the WDC. The priority for these cases is getting items returned to owners during the competition. The WDC will generally not apply disciplinary action unless a competitor has been caught stealing after a previous warning or all of the following conditions are met:
-  * There is definitive proof of the accused person committing theft (such as a witness seeing them steal, CCTV footage, admission of guilt, etc.).
-  * The suspect has been contacted and asked to return missing items.
-  * The suspect has been unresponsive/uncooperative and has not returned missing items.
-* If you receive reports that lead you to believe that there has been theft, then do your best to resolve the situation during the competition if possible.
-  * Make an announcement, giving a description of missing items and advising competitors to give items to an organizer or Delegate if they are found. In past incidents, competitors have returned missing items after similar announcements.
-  * Ask competitors if they witnessed anything that could be related to the theft (e.g. someone standing next to a table where puzzles went missing or seeing someone playing with a puzzle that went missing).
-  * Talk to any suspects and ask direct questions. If they confess and return any missing items, thank them for their honesty and warn them to avoid stealing in the future if they deliberately took the item. Keep the age and maturity of the individual in mind.
-  * Ask the venue for security camera footage to identify potential suspects if possible.
-* If puzzles were not returned at the competition, but you believe you know who may have stolen them then contact them and ask for the puzzles to be returned. Contact the WDC if the individual is not responsive or is unwilling to return the puzzles.
-* Report incidents of theft to the WDC so that we can note down competitors names in case there are repeated cases of theft in the future.
+- The WIC’s preference is for theft (such as stealing puzzles) to be investigated and resolved locally when possible, as the local Delegates can often resolve these cases better than the WIC. The priority for these cases is getting items returned to owners during the competition. The WIC will generally not apply disciplinary action unless a competitor has been caught stealing after a previous warning or all of the following conditions are met:
```

Comment:
The apostrophe in WIC's here is not the standard Unicode apostrophe character. It might be best to replace all instances of this alternate character with the standard one (').